core/templates/base-components/base-content.directive.html

```diff
@@ -111,5 +111,11 @@
     color: #000;
     font-size: 25px;
     padding: 10px;
+
+    @media (min-width: 768px) {
+      .oppia-base-content-pre-logo-container {
+        width: 100%;
+    }
+    }
```

Comment:
There should be some extra spacing here.

Reply:
These changes were not related to the about-page. So removed them.